src/debuginfo.cpp

```diff
@@ -343,7 +343,9 @@ class JuliaJITEventListener: public JITEventListener
         for (const object::SymbolRef &sym_iter : debugObj.symbols()) {
             StringRef sName;
 #ifdef LLVM37
-            sName = sym_iter.getName().get();
+            auto sNameOrError = sym_iter.getName();
+            assert(sNameOrError);
```

Comment:
You are right, thank you, I overlooked that. So there won't be issues with the release build.